Automattic/jetpack
# Nav Unification: Remove Hover Effect at Top of the Sidebar

Fixes Automattic/wp-calypso#51835

#### Changes proposed in this Pull Request:

* Removes the hover effect introduced here: https://core.trac.wordpress.org/changeset/49961

#### Jetpack product discussion


* Some discussion in Automattic/wp-calypso#51835

#### Does this pull request change what data or activity we track or use?


* Nope

#### Testing instructions:






* Hover over the "Browse Sites" and site card in the Sidebar 
* Verify that no border appears

| Before                                                                                                                                                                | After                                                                                                                                                                 |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| <img width="259" alt="Screenshot 2021-04-12 at 19 47 37" src="https://user-images.githubusercontent.com/43215253/114446089-be18e880-9bc8-11eb-8b26-ffa64b05bbb2.png"> | <img width="262" alt="Screenshot 2021-04-12 at 19 47 28" src="https://user-images.githubusercontent.com/43215253/114446085-bd805200-9bc8-11eb-8d5c-c697f4ed4049.png"> |
